cibuild.cmd

```diff
@@ -78,6 +79,12 @@ if defined TestDeterminism (
     exit /b 0
 )
 
+if defined TestPerfCorrectness (
+    msbuild %MSBuildAdditionalCommandLineArgs% Roslyn.sln /p:Configuration=%BuildConfiguration% || goto :BuildFailed
+    .\Binaries\%BuildConfiguration%\Roslyn.Test.Performance.Runner.exe
```

Comment:
Shouldn't htis have `|| goto :BuildFailed`?  As is it looks like this will successfully complete 100% of the time.